tests/views/test_collection_exercise.py

```diff
@@ -269,8 +277,39 @@ def test_upload_sample(self, mock_request):
             "ingestDateTime": "2017-11-06T14:02:24.203+0000"
         }
 
+        survey_data = {
+            "id": "af6ddd8f-7bd0-4c51-b879-ff4b367461c5"
+        }
+
+        exercise_data = [{
```

Comment:
survey_data and exercise_data should probably get loaded in from one of the json files in tests/test_data/ directory like at the top of this file.

Reply:
:+1: anything thats large like this should be in it's own file
We can probably clean up how we pull in our test data across the python repos